partiql-conformance-tests/build.rs

```diff
@@ -46,6 +46,18 @@ fn main() -> io::Result<()> {
             .unwrap_or_else(|error| panic!("Failure when writing to file: {:?}", error));
     }
 
-    dir_to_mods(tests_path);
+    let sub_tests_dir = "partiql_tests";
+    let sub_tests_path = Path::new(sub_tests_dir);
+    dir_to_mods(&tests_path.join(sub_tests_path));
+    File::create(&tests_path.join("mod.rs"))
+        .expect("mod.rs created in root test folder")
+        .write_all(
+            format!(
+                "#[cfg(feature = \"conformance_test\")]\n#[rustfmt::skip]\nmod {};\n",
+                sub_tests_dir
+            )
+            .as_bytes(),
+        )
+        .unwrap_or_else(|error| panic!("Failure when writing to file: {:?}", error));
```

Comment:
`unwrap_or_else` with a panic inside can be replaced with `expect`: https://doc.rust-lang.org/std/result/enum.Result.html#method.expect

Reply:
Changed to use `expect` at a few places